pkg/security/probe/probe_kernel_file_windows.go

```diff
@@ -77,8 +77,24 @@ type createArgs struct {
 	fileName         string
 }
 
-func parseCreateArgs(e *etw.DDEventRecord) (*createArgs, error) {
-	ca := &createArgs{}
+type createNewFileArgs createHandleArgs
```

Comment:
nit : could we use an alias here ?

Reply:
as discussed in slack.  don't want an alias, want an actually different type, so we can use the same code to decode the same underlying structure, but use type switch to determine what type of message it actually is.